tests/unit/states/test_xml.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+'''
+Test cases for xml state
+'''
+# Import Python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import skipIf, TestCase
+from tests.support.mock import (
+    NO_MOCK,
+    NO_MOCK_REASON,
```

Comment:
Ah - apparently that *other* change got merged in. There's no such complaint as `NO_MOCK` and `NO_MOCK_REASON` anymore. Any usages of these two in the PR should be removed.

Reply:
removed!